lib/ansible/plugins/filter/encryption.py

```diff
@@ -17,7 +17,7 @@
 display = Display()
 
 
-def do_vault(data, secret, salt=None, vaultid='filter_default', wrap_object=False):
+def do_vault(data, secret, salt=None, vault_id='filter_default', wrap_object=False):
```

Comment:
Isn't this a breaking change, since every use which uses the `vaultid` option for this filter at the moment will stop working after this is merged?

Same for `do_unvault`.

Reply:
yes .. but vaultid was not documented ... vault_id was